src/classes/SftpSessionHandler.ts

```diff
@@ -114,7 +133,8 @@ export class SftpSessionHandler {
     })
       .then(async (response) => {
         const data = await response.buffer();
-        logger.debug('Response: Data', { reqId, data });
+        logger.verbose('Response: Data', { reqId, data });
```

Comment:
Whoops!  the whole point is NOT posting data in verbose mode, need to remove this